openssl-dynamic/src/main/java/io/netty/internal/tcnative/CertificateVerifier.java

```diff
@@ -25,73 +29,155 @@
+    public static final int X509_V_ERR_UNHANDLED_CRITICAL_CRL_EXTENSION = x509vErrUnhandledCriticalCrlExtension();
+    public static final int X509_V_ERR_INVALID_NON_CA = x509vErrInvalidNonCa();
+    public static final int X509_V_ERR_PROXY_PATH_LENGTH_EXCEEDED = x509vErrProxyPathLengthExceeded();
+    public static final int X509_V_ERR_KEYUSAGE_NO_DIGITAL_SIGNATURE = x509vErrKeyUsageNoDigitalSignature();
+    public static final int X509_V_ERR_PROXY_CERTIFICATES_NOT_ALLOWED = x509vErrProxyCertificatesNotAllowed();
+    public static final int X509_V_ERR_INVALID_EXTENSION = x509vErrInvalidExtension();
+    public static final int X509_V_ERR_INVALID_POLICY_EXTENSION = x509vErrInvalidPolicyExtension();
+    public static final int X509_V_ERR_NO_EXPLICIT_POLICY = x509vErrNoExplicitPolicy();
+    public static final int X509_V_ERR_DIFFERENT_CRL_SCOPE = x509vErrDifferntCrlScope();
+    public static final int X509_V_ERR_UNSUPPORTED_EXTENSION_FEATURE = x509vErrUnsupportedExtensionFeature();
+    public static final int X509_V_ERR_UNNESTED_RESOURCE = x509vErrUnnestedResource();
+    public static final int X509_V_ERR_PERMITTED_VIOLATION = x509vErrPermittedViolation();
+    public static final int X509_V_ERR_EXCLUDED_VIOLATION  = x509vErrExcludedViolation();
+    public static final int X509_V_ERR_SUBTREE_MINMAX = x509vErrSubtreeMinMax();
+    public static final int X509_V_ERR_APPLICATION_VERIFICATION = x509vErrApplicationVerification();
+    public static final int X509_V_ERR_UNSUPPORTED_CONSTRAINT_TYPE = x509vErrUnsupportedConstraintType();
+    public static final int X509_V_ERR_UNSUPPORTED_CONSTRAINT_SYNTAX = x509vErrUnsupportedConstraintSyntax();
+    public static final int X509_V_ERR_UNSUPPORTED_NAME_SYNTAX = x509vErrUnsupportedNameSyntax();
+    public static final int X509_V_ERR_CRL_PATH_VALIDATION_ERROR = x509vErrCrlPathValidationError();
+    public static final int X509_V_ERR_PATH_LOOP = x509vErrPathLoop();
+    public static final int X509_V_ERR_SUITE_B_INVALID_VERSION = x509vErrSuiteBInvalidVersion();
+    public static final int X509_V_ERR_SUITE_B_INVALID_ALGORITHM = x509vErrSuiteBInvalidAlgorithm();
+    public static final int X509_V_ERR_SUITE_B_INVALID_CURVE = x509vErrSuiteBInvalidCurve();
+    public static final int X509_V_ERR_SUITE_B_INVALID_SIGNATURE_ALGORITHM = x509vErrSuiteBInvalidSignatureAlgorithm();
+    public static final int X509_V_ERR_SUITE_B_LOS_NOT_ALLOWED = x509vErrSuiteBLosNotAllowed();
+    public static final int X509_V_ERR_SUITE_B_CANNOT_SIGN_P_384_WITH_P_256 = x509vErrSuiteBCannotSignP384WithP256();
+    public static final int X509_V_ERR_HOSTNAME_MISMATCH = x509vErrHostnameMismatch();
+    public static final int X509_V_ERR_EMAIL_MISMATCH = x509vErrEmailMismatch();
+    public static final int X509_V_ERR_IP_ADDRESS_MISMATCH = x509vErrIpAddressMismatch();
+    public static final int X509_V_ERR_DANE_NO_MATCH = x509vErrDaneNoMatch();
+
+    private static final Set<Integer> ERRORS;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AModifiersOrderCheck)